malApp.R

```diff
@@ -15,10 +15,25 @@ min_date<- substr(min_date[1,"min"], 1, 10)
 max_date <- read.csv("/home/mkhuphuli/hello/credentials.csv", header=TRUE) %>%
```

Comment:
don't put your full directry here
just add a relative one: `///hom/dir/to/file/`